containers/cpp-mariadb/.devcontainer/install-mariadb.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -e
+
+. /etc/os-release
+
+MARIADB_REPO_SETUP=mariadb_repo_setup
+TMP_DIR=$(mktemp -d -t maria-XXXXXXXXXX)
+MARIADB_CONNECTOR=""
+
+function cleanup {
+    cd /
+    rm -rf ${TMP_DIR}/${MARIADB_REPO_SETUP}
+    rm -rf ${TMP_DIR}/${MARIADB_CONNECTOR}*
+}
+trap cleanup EXIT
+
+#Set up external repository and install C Connector
+cd ${TMP_DIR}
+curl -Ls https://downloads.mariadb.com/MariaDB/${MARIADB_REPO_SETUP} -o ${MARIADB_REPO_SETUP}
+echo "fd3f41eefff54ce144c932100f9e0f9b1d181e0edd86a6f6b8f2a0212100c32c  ${MARIADB_REPO_SETUP}" | sha256sum -c -
+chmod +x ${MARIADB_REPO_SETUP}
+./${MARIADB_REPO_SETUP} --mariadb-server-version="mariadb-10.6"
+apt install -y libmariadb3 libmariadb-dev
+
+#Depending on the OS, install different C++ connectors
+OSURL=""
+OSTAG=""
+case $ID in
```

Comment:
This `case` statement (lines 28 to 62) could potentially be extracted into a function that exports `OSURL` and
`OSTAG` to make the code more readable and cleaner.